tests/ui/associated-type-bounds/overlaping-bound-suggestion.stderr

```diff
@@ -6,6 +6,11 @@ LL |     inner: <IntoIterator<Item: IntoIterator<Item: >>::IntoIterator as Item>
    |             |                  |
    |             |                  associated types `Item`, `IntoIter` must be specified
    |             associated types `Item`, `IntoIter` must be specified
+  --> $SRC_DIR/core/src/iter/traits/collect.rs:LL:COL
+   |
+   = note: `IntoIterator::Item` shadowed here
+   |
+   = note: `IntoIterator::Item` shadowed here
```

Comment:
I don't understand what this error is trying to say, `IntoIterator` has only one `type Item;` and no base traits.

Reply:
This happened because I didn't check that the shadowed associated type actually comes from a different trait.
It should be fixed now